src/features/reader/modules/miniflux_end_of_book.lua

```diff
@@ -176,6 +176,8 @@ function MinifluxEndOfBook:showDialog(entry_info)
                         if ReaderUI.instance then
                             ReaderUI.instance:onClose()
                         end
+                        -- Return to the browser view where the user was before opening this entry
+                        self:returnToBrowser()
```

Comment:
It would also be nice to have another button "Read entries" that just returns to the browser.

Edit: just saw your comment about chainging the miniflux folder button to return to the browser and that would be much better